dubbo-compatible/src/main/java/com/alibaba/dubbo/common/URL.java

```diff
@@ -26,6 +26,10 @@
 @Deprecated
 public class URL extends org.apache.dubbo.common.URL {
 
+    protected URL() {
```

Comment:
seems like public is better.

Reply:
@carryxyh  I don't think so. Because a good URL must have basic information, such as protocol, host, port.

Reply:
emmm...
Maybe u are right. Seems like a protected constructor can fix this problem, too.